en/concepts/scene/light/additive-per-pixel-lights.md

```diff
@@ -22,15 +22,15 @@ As shown in the image above, the first rendering is the lighting of **Directiona
 
 ![main light pass](additivelights/pass1.png)
 
-The second rendering is the lighting of **Spot Light 1**:
+The second rendering is the lighting of **Spotlight 1**:
 
 ![ForwardAdd pass](additivelights/pass2.png)
 
-The third rendering is the lighting of **Spot Light 2**:
+The third rendering is the lighting of **Spotlight 2**:
 
 ![ForwardAdd pass](additivelights/pass3.png)
 
-This type of rendering is the **Forward-Pipeline** that supports multiple lighting models, and **Forward** generally consists of two passes:
+This type of rendering is the **Forward-Pipeline** that supports multiple lighting models, and **Forward** generally consists of two Passes:
```

Comment:
why is Passes capitalized?

Reply:
Maybe:

```suggestion
This type of rendering is the **Forward-Pipeline** that supports multiple lighting models, and **Forward** generally consists of two `Pass`:
```

Reply:
You can read it in context to understand it.

Reply:
I think backticks helps demonstrate it is a technical term versus what we use "passes" for in everyday language.